TASVideos/Pages/Account/Register.cshtml.cs

```diff
@@ -133,48 +133,28 @@ public async Task<IActionResult> OnPost()
 				var result = await _userManager.CreateAsync(user, Password);
 				if (result.Succeeded)
 				{
-					await AddStandardRoles(user.Id);
-					await _userManager.AddUserPermissionsToClaims(user);
+					var code = await _userManager.GenerateEmailConfirmationTokenAsync(user);
```

Comment:
can we call this variable "token" or "confirmationToken" to be more consistent with the method being called?